Project.toml

```diff
@@ -1,11 +1,12 @@
 name = "MatrixFactorizations"
 uuid = "a3b82374-2e81-5b9e-98ce-41277c0e4c87"
-version = "0.6.1"
+version = "0.6.2-DEV"
```

Comment:
Don't bother with -dev as I'll tag as soon as merged.

It should be `"0.7" as you export new names